website/server/controllers/top-level/dataexport.js

```diff
@@ -25,12 +25,27 @@ const BASE_URL = nconf.get('BASE_URL');
 let api = {};
 
 /**
+ * @apiDefine authError
```

Comment:
Let's not use this in the documentation for each route. We should instead have some info in the header of the API doc about authorization requirements. (not for this PR)